src/@next/Banner/Banner.tsx

```diff
@@ -123,12 +123,16 @@ export const Banner = React.forwardRef<HTMLDivElement, BannerProps>(
             <Icon name={iconNameValue} />
           </StyledFixedIconWrapper>
         )}
-        <StyledFixedBannerContentContainer data-noicon={showIcon ? '' : 'true'}>
+        <StyledFixedBannerContentContainer
+          data-show-icon={showIcon}
+          data-has-action={action || secondaryAction ? 'true' : ''}
+          data-dismissable={dismissable}
+        >
           <Typography as="div" variant="body1">
```

Comment:
hi @michac789 according to Rebecca, the text style here should be changed to body2